course-definition.yml

```
@@ -408,3 +417,56 @@ stages:
       **Example**:
 
       - `(cat|dog)` should match "dog" and "cat", but not "apple".
+
+  # Backreferences
+
+  - slug: "br-basic"
```

```suggestion
  - slug: "backreferences-single"
```

(These will show up in URLs someday, so let's try to make them non-cryptic)